lib/Controller/DelegationController.php

```diff
@@ -47,7 +47,7 @@ public function __construct($AppName,
 		AuthorizedGroupService $authorizedGroupService) {
 		parent::__construct($AppName, $request);
 		$this->config = $config;
-		$this->groupManager = $groupManager;
+		$this->groupManager =  $groupManager;
```

Comment:
Hmm, try composer cs:fix

Reply:
I fixed it in this commit : https://github.com/nextcloud/groupfolders/pull/2072/commits/8c3401aeea04d30553c13a7e6ba3f5ce902a9e15

I used this command : `./vendor/php-cs-fixer .`